service-front/module/Application/src/Validators/PassportDateValidatorCp.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Validators;
+
+use Laminas\Validator\AbstractValidator;
+
+class PassportDateValidatorCp extends AbstractValidator
+{
+    public const PASSPORT_DATE = 'passport_date';
+
+    public const EXPIRY_ALLOWANCE = '+3 year';
+
+    protected array $messageTemplates = [
+        self::PASSPORT_DATE => 'The passport needs to be no more than 3 years out of date. Check the expiry date 
+        and change to Yes, or try a different method',
+    ];
```

Comment:
Is this error message ever displayed? It looks like the template hard codes a valid/invalid response rather than using it?